unlock-app/src/components/interface/checkout/main/Password.tsx

```diff
@@ -86,7 +86,10 @@ export function Password({ injectedProvider, checkoutService }: Props) {
               validate: (password: string) => {
                 const { address } = getEthersWalletFromPassword(password) ?? {}
                 // check if password match
-                return passwordSigner === address || 'Wrong password...'
+                if (passwordSigner && passwordSigner !== address) {
```

Comment:
If the `passwordSigner` is not set, let's assume the password is fine. No need to do anything as the transaction will just pass.